project/CSSGen.scala

```diff
@@ -3,8 +3,16 @@ import scala.sys.process._
 
 object CSSGen {
 
-  def genCssProp(name: String): String =
-    s"""  def `$name`(value: String): Style = Style("$name", value)"""
+  def genCssProp(name: String): String = {
+    val prop = s"""  def `$name`(value: String): Style = Style("$name", value)"""
+    if (!prop.contains("-") || prop.contains('@')) { prop }
```

Comment:
I think you want to be testing `name` in the predicate, not `prop`, so:
`if (!name.contains("-") || name.contains('@')) { prop }`

However, you also don't need the `!name.contains("-")` test, because spliting a string without a hyphen is just a list of one element, and should work (I think).

Reply:
Yeah, name was definitely what I meant there. checking if it contains the hyphen is just to avoid double declaration of the method.